app/scripts/directives/mobileClientServiceAssociations.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+(function() {
+  angular.module('openshiftConsole').component('mobileClientServiceAssociations', {
+    controller: [
+      '$filter',
+      'APIService',
+      'DataService',
+      'MobileClientsService',
+      'NotificationsService',
+      MobileClientServiceAssociationsCtrl
+    ],
+    bindings: {
+      project: '<',
+      mobileClient: '<',
+      serviceClasses: '<'
+    },
+    templateUrl: 'views/directives/mobile-client-service-associations.html'
+  });
+
+  function MobileClientServiceAssociationsCtrl(
+    $filter,
+    APIService,
+    DataService,
+    MobileClientsService,
+    NotificationsService
+  ) {
+
+    var ctrl = this;
+    var context = { namespace: ctrl.project };
+    var watches = [];
+    var getErrorDetails = $filter('getErrorDetails');
+    var isServiceInstanceReady = $filter('isServiceInstanceReady');
+    var isMobileService = $filter('isMobileService');
+
+    //TODO: Move to service or use APIService.getPreferredService
+    var mobileclientVersion = {
```

Comment:
You can use
```
var mobileclientVersion = APIService.kindToResourceGroupVersion({group: "mobile.k8s.io", kind: "mobileclients"});
```
for the alpha stuff.

Reply:
@witmicko has this open too? https://github.com/openshift/origin-web-common/pull/320
Whats the significance of the alpha?